context/app/static/js/components/error/ErrorBody/ErrorBody.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { LightBlueLink } from 'js/shared-styles/Links';
+
+const LoginLink = () => <LightBlueLink href="/login">login</LightBlueLink>;
+const HelpEmailLink = () => (
+  <LightBlueLink href="mailto:help@hubmapconsortium.org">help@hubmapconsortium.org</LightBlueLink>
+);
+
+function ErrorBody({ errorCode, isAuthenticated, isGlobus401, isMaintenancePage }) {
+  if (isMaintenancePage) {
+    return (
+      <>
+        While the portal is under maintenance, visit the{' '}
+        <LightBlueLink href="https://hubmapconsortium.org/" target="_blank" rel="noopener noreferrer">
+          HuBMAP Consortium
+        </LightBlueLink>{' '}
+        website.
+      </>
+    );
+  }
+
+  if (errorCode === 401 && isGlobus401) {
+    return (
+      <>
+        Your credentials have expired. Please <LoginLink /> again.
+      </>
+    );
+  }
+
+  if (errorCode === 401) {
+    return (
+      <>
+        You have not been added to the HuBMAP Group on Globus. Request access at <HelpEmailLink />.
+      </>
+    );
+  }
+
+  if (errorCode === 403 && isAuthenticated) {
+    return (
+      <>
+        You may not have access to this resource. Request access at <HelpEmailLink />.
+      </>
+    );
+  }
+
+  if (errorCode === 403) {
+    return (
+      <>
+        You may not have access to this resource or you need to <LoginLink />.
+      </>
+    );
+  }
+
+  if (errorCode === 404) {
+    return (
+      <>
+        If this page should exist, submit a bug report to <HelpEmailLink />.
```

Comment:
I think the messaging can be more confident... If someone just mis-copied and pasted the URL, rather than double checking, this messages will encourage them to hit the help desk.

Reply:
@tsliaw

Reply:
I think the error message is fine for now, but also feel free to follow up on this issue in slack or the meeting agenda to get more feedback.